flow/connectors/bigquery/bigquery.go

```diff
@@ -621,9 +621,9 @@ func (c *BigQueryConnector) SetupNormalizedTable(
 		genericColType := column.Type
 		if genericColType == "numeric" {
 			precision, scale := numeric.ParseNumericTypmod(column.TypeModifier)
```

Comment:
`ParseNumericTypmod` could take `WarehouseNumericCompatibility` to do -1/valid checks